packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -323,7 +323,7 @@ class TextPainter {
     _needsLayout = true;
   }
 
-  /// {@macro flutter.dart:ui.text.TextWidthBasis}
+  /// Defines how to measure the width of the rendered text.
```

Comment:
Since we are repeating the the same documentation across text_painter selectable_text and text, I think we can define the template here and use macro for selectable_text and text?

Reply:
Good call, will do.